modules/apps/27-interchain-accounts/host/keeper/msg_server_test.go

```diff
@@ -0,0 +1,56 @@
+package keeper_test
+
+import (
+	"github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/host/keeper"
+	"github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/host/types"
+)
+
+func (suite *KeeperTestSuite) TestUpdateParams() {
+	msg := types.MsgUpdateParams{}
+
+	testCases := []struct {
+		name     string
+		malleate func(authority string)
+		expPass  bool
+	}{
+		{
+			"invalid authority address",
+			func(authority string) {
+				msg.Authority = "authority"
+				msg.Params = types.DefaultParams()
+			},
+			false,
+		},
+		{
+			"success",
```

Comment:
Nit: by convention we usually place the success cases first.

Reply:
done